assets/app/views/directives/events.html

```diff
@@ -0,0 +1,36 @@
+<div class="middle-content">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-12">
```

Comment:
Actually I don't think the directive should have container-fluid -> row -> col-md-12 at all. Let the including page decide layout. I'd only have the table element in the directive view.

The screenshots had some left and right margin to the sides of the events table that looked odd. This might be why.